### PR TITLE
Test/#108/video box component

### LINKED ISSUE #108: (SP: 1) Write unit test for "VideoBox" component
### Component unit test
Unit test for "**VideoBox**" component
Scenaries descriptions:
- [ ] Video should be in the component

[Link to component](https://github.com/Space2Study-UA-1125/frontEnd/blob/develop/src/components/video-box/VideoBox.jsx)

Current coverage:
Image

### PR DESCRIPTION
**Pull request by OliaHrytsak**
Unit test for "VideoBox" component

image

image

### CODE REVIEW

**Review by VictoriaRodik — APPROVED** [review]

File: tests/unit/components/video-box/VideoBox.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { describe } from 'vitest'
+import { screen, render } from '@testing-library/react'
+import VideoBox from '~/components/video-box/VideoBox.jsx'
+
+describe('VideoBox component test', () => {
+  beforeEach(() => {
+    render(<VideoBox />)
```
Comment (iva-stasia) [inline tests/unit/components/video-box/VideoBox.spec.jsx:6]: I guess you can improve your test a bit by adding a video path test to make sure it's the right video. For this, you need to pass the corresponding property to the component and then check if the video path matches the one passed.
```javascript
render(<VideoBox video={videoSrc} />)
```

**Review by iva-stasia — APPROVED** [review]

**Review by holus-bolus — APPROVED** [review]

**Review by Wiret02 — APPROVED** [review]

File: tests/unit/components/video-box/VideoBox.spec.jsx (hunk 1 above)
Comment (OliaHrytsak) [reply tests/unit/components/video-box/VideoBox.spec.jsx:6]: Thank you for your comment
It was necessary to add
I just done it

**Review by ashfox21 — APPROVED** [review]

File: tests/unit/components/video-box/VideoBox.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { screen, render } from '@testing-library/react'
+import VideoBox from '~/components/video-box/VideoBox'
+
+describe('VideoBox component test', () => {
+  const video = '~/assets/img/guest-home-page/videoImg.png'
```
Comment (vlad-khrychov) [inline tests/unit/components/video-box/VideoBox.spec.jsx]: For what purpose you use this constant? On 22th line you shadow it.

**Review by vlad-khrychov — APPROVED** [review]
Did you try create test with snapshots?

File: tests/unit/components/video-box/VideoBox.spec.jsx (hunk 2 above)
Comment (OliaHrytsak) [reply tests/unit/components/video-box/VideoBox.spec.jsx]: I've just remove this constant

**Pull request merged**